core/src/main/java/org/apache/carbondata/core/datamap/TableDataMap.java

```diff
@@ -526,4 +532,29 @@ public long getRowCount(List<Segment> allsegments, final List<PartitionSpec> par
     return totalRowCount;
   }
 
+  /**
+   * Method to prune the segments based on task min/max values
+   *
+   * @param segments
```

Comment:
remove it if you are not writing description

Reply:
done